components/notification/NotificationCard.vue

```diff
@@ -15,7 +15,6 @@ const { notification } = defineProps<{
           ps-3 pe-4 inset-is-0
           rounded-ie-be-3
           py-3 bg-base top-0
-          :lang="notification.status?.language ?? undefined"
```

Comment:
Could you explain why this got removed? also below

Reply:
A `follow` notification never has a `status`, same for the others were it was removed. The types in masto v6 may better reflect that and it was erroring out after the upgrade.